### PR TITLE
Document #341.

### PR DESCRIPTION
**Pull request by fisx**
This does not fix this issue, but sheds some light on it in case it pops up again in the future.

### CODE REVIEW

File: services/gundeck/test/unit/Native.hs [hunk 1]
```diff
@@ -32,8 +32,10 @@ import qualified OpenSSL.EVP.Digest      as EVP
 tests :: TestTree
 tests = testGroup "Native"
     [ testProperty "serialise/ok" $
+        -- this may fail sporadically, but that's not a production issue.
+        -- see <https://github.com/wireapp/wire-server/issues/341>.
         forAll genTransport serialiseOkProp
-    , testProperty "serialise/size-limit" $
+      testProperty "serialise/size-limit" $
```
Comment (neongreen) [inline services/gundeck/test/unit/Native.hs]: Accidental?

File: services/gundeck/test/unit/Native.hs [hunk 2]
```diff
@@ -63,7 +65,8 @@ serialiseOkProp t = ioProperty $ do
             plain <- EVP.cipherBS c (encKeyBytes encKey) iv EVP.Decrypt dat'
             let n' = plainNotif <$> decodeStrict' plain
             return $ n' == Just n && mac == cipherMac p
-    return $ equalTransport && equalNotif
+    let debugInfo = (t, a, n, {- c, d, m, -} r, sn, equalTransport, equalNotif)
```
Comment (neongreen) [inline services/gundeck/test/unit/Native.hs]: Consider adding variable names, e.g.

```haskell
let debugInfo = intercalate ", " $ zipWith (\name var -> name ++ "=" ++ var)
      [ ("something something", t)
      , ("dunno what", a)
      , ... ]
return $ counterexample debugInfo (equalTransport && equalNotif)
```

Comment (fisx) [reply services/gundeck/test/unit/Native.hs]: i'd rather not spend the time.  this only happens if a test case fails, in which case we need to take a closer look at the source anyway.

File: services/gundeck/test/unit/Native.hs (hunk 1 above)
Comment (fisx) [reply services/gundeck/test/unit/Native.hs]: yes, that's broken.  will fix.  thanks!

**Comment by tiago-loureiro** [comment]
@fisx I updated the branch to get the flaky limit test in

**Comment by neongreen** [comment]
Let's merge this in?

**Pull request merged**